element_calcium_imaging/scan.py

```diff
@@ -180,6 +188,7 @@ class ScanInfo(dj.Imported):
     z=null               : float     # (um) ScanImage's 0 point in the motor coordinate system
     fps                  : float     # (Hz) frames per second - Volumetric Scan Rate 
     bidirectional        : boolean   # true = bidirectional scanning
+    bidirectional_z=null : booleaen  # true = bidirectional z-stack for PrairieView
```

Comment:
Hm... This worked? So probably did not get tested.
```suggestion
    bidirectional_z=null : boolean  # true = bidirectional z-stack for PrairieView
```